src/java_bytecode/java_entry_point.cpp

```diff
@@ -46,32 +47,33 @@ Function: create_initialize
 
 \*******************************************************************/
 
-namespace {
```

Comment:
The linter says no namespaces, I'm not sure if this includes anon namespaces, and if so what should be done about this function. Probably should end up in a class per this refactor suggestion: https://github.com/diffblue/cbmc/pull/338#issuecomment-265497399